### PR TITLE
[16][ADD] rma_category

### PR DESCRIPTION
**Pull request by florian-dacosta**
Add the possibility to add one or more categories on RMA

### CODE REVIEW

File: rma_category/README.rst [hunk 1]
```diff
@@ -0,0 +1,37 @@
+==============
```
Comment (AaronHForgeFlow) [inline rma_category/README.rst]: I am afraid this will not be automatically changed so it is best to change it manually

**Review by AaronHForgeFlow — COMMENTED** [review]
LGTM, I would appreciate a test, but as long as there is no logic we can skip that.

**Comment by florian-dacosta** [comment]
> LGTM, I would appreciate a test, but as long as there is no logic we can skip that.

Indeed, there are not logic at all, just a way to classify the RMA with a M2O. I would not know which test I could do for now !

**Review by AaronHForgeFlow — COMMENTED** [review]
LGTM including the fix in RMA view. Can you please put the RMA change in a separate commit? Also, the readme won't be created automatically by the content in the readme folder, so I am afraid you have to do it :'(

**Comment by florian-dacosta** [comment]
> LGTM including the fix in RMA view. Can you please put the RMA change in a separate commit? Also, the readme won't be created automatically by the content in the readme folder, so I am afraid you have to do it :'(

I did a separate commit.
About the README, I made the change already 2 days ago.

**Review by AaronHForgeFlow — APPROVED** [review]
Thank you @florian-dacosta

**Comment by florian-dacosta** [comment]
Well, pre-commit fails suddenly, I have no idea why...

**Comment by AaronHForgeFlow** [comment]
I will check that, perhaps we have to update pre-commit version

**Comment by AaronHForgeFlow** [comment]
It seems related to change in python version in github actions:

https://github.com/OCA/oca-addons-repo-template/pull/214

**Comment by AaronHForgeFlow** [comment]
@florian-dacosta  if you rebase it should be green now.

**Comment by florian-dacosta** [comment]
Great, thanks !

**Pull request merged**